src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BindLoweringPass.cs

```diff
@@ -230,20 +240,19 @@ private void RewriteUsage(TagHelperIntermediateNode node, int index, ComponentAt
             // BindMethods.SetValueHandler(__value => <code> = __value, <code>) OR
             // BindMethods.SetValueHandler(__value => <code> = __value, <code>, <format>)
             //
-            // For now, the way this is done isn't debuggable. But since the expression
-            // passed here must be an LValue, it's probably not important.
+            // Note that the linemappings here are applied to the value attribute, not the change attribute.
```

Comment:
We might get feedback on this and it might be worth rethinking. Currently the change handler will be written out without any line mappings, which means that it's not debuggable. I don't think it's a big concern for bind since you can't really write a lambda here.

Reply:
OK. We'll have to see how it works in practice when we have debugging. I'm sure it's fine for now.